setup.cfg

```diff
@@ -34,6 +34,8 @@ install_requires =
     scipy
     particletools
     particle
+    tqdm
```

Comment:
tqdm is not an install requirement anymore, right?

Reply:
Yes, it is not a requirement and it is removed.